docs/env.md

```diff
@@ -106,6 +106,45 @@ export ANNOUNCEMENT="Hello! I'm {USERNAME} and I'm an SSH bot! I'm currently lis
 export ANNOUNCEMENT="Hello! I'm {USERNAME} and I'm an SSH bot! Being in {CURRENT_TEAM} will grant you SSH access to certain servers. Reach out to @dworken for more information."
 ```
 
+## Two Man Realms
+
+It is possible to configure the SSH CA bot to require approval prior to granting access. For example, one could require
+that in order to use kssh with servers in the `team.ssh.root_everywhere` realm two other people must approve the request. 
+Approvals are done by reacting with a :+1: emoji to the message inside of Keybase chat. 
+
+In order to configure this, there are 3 environment variables that can be used:
+
+```
+export TWO_MAN_TEAMS="team.ssh.root_everywhere, team.ssh.ci"
+export TWO_MAN_APPROVERS="username0, username1, username2" 
+export TWO_MAN_APPROVER_COUNT="2"
```

Comment:
`TWO_MAN_` is a little confusing since the number of approvers is variable. it sounds to me like this is an M of N control policy. can we call it something like 
```
CTRL_POLICY_1_MOFN_TEAMS="team.ssh.root_everywhere, team.ssh.ci"
CTRL_POLICY_1_M=2
CTRL_POLICY_1_N="username0, username1, username2"
```
is it possible we could want multiple control policies managed by a single bot? if so, this might need to be more complex. that's why i shoved a 1 in there. maybe not necessary right now. i dunno.

Reply:
Done! I think for now it isn't necessary to have the 1 in there since that seems like a pretty advanced feature and probably beyond the scope of this project.